db/migrate/20171001043440_create_games.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGames < ActiveRecord::Migration[5.0]
+  def change
+    create_table :games do |t|
+      t.integer :id
```

Comment:
I don't think you need to explicitly add `id` here. Rails does it for you with create_table